ViAn/GUI/projectwidget.cpp

```diff
@@ -582,14 +583,26 @@ void ProjectWidget::create_folder_item() {
  */
 void ProjectWidget::save_project() {
     if (m_proj == nullptr ) return;
+    QDir dir(QString::fromStdString(m_proj->get_dir()));
```

Comment:
This popup is triggered on each save. It should only happen when the project has never been saved to that location before.